python/ooi_data_explorations/common.py

```diff
@@ -394,14 +405,153 @@ def get_annotations(site, node, sensor):
     :param sensor: Sensor name to query
     :return:
     """
-    r = SESSION.get(BASE_URL + ANNO_URL + 'find?beginDT=0&refdes=' + site + '-' + node + '-' + sensor,
-                    auth=(AUTH[0], AUTH[2]))
+    r = SESSION.get(BASE_URL + ANNO_URL + 'find?beginDT=0&refdes=' + site + '-'
+                    + node + '-' + sensor, auth=(AUTH[0], AUTH[2]))
     if r.status_code == requests.codes.ok:
         return r.json()
     else:
         return None
 
 
+def add_annotation_qc_flags(ds, annotations):
+    """Add the annotation qc flags to a dataset as a data variable.
```

Comment:
Please change the docstring formatting to match all the other functions. The code uses the PEP 287 reStructured Text docstring format (https://www.python.org/dev/peps/pep-0287/).